.github/workflows/lint-and-template-check.yml

```diff
@@ -0,0 +1,60 @@
+name: Lint and PR Template Check
+
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - opened
+      - reopened
+      - synchronize
+      - edited
+  pull_request_review_comment:
+    types:
+      - created
+env:
+  LABELS: ${{ join( github.event.pull_request.labels.*.name, ' ' ) }}
+
+jobs:
+  lint-and-template-check:
+    runs-on: ubuntu-latest
+    name: Lint and Template Check
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/hydrogen'
+      
+      - name: Install dependencies
+        run: npm install --no-audit --progress=false
+      
+      - name: Run lint
+        run: npm run lint
```

Comment:
don't we already do this?

Reply:
https://github.com/akhenry/openmct-yamcs/blob/5b7b96159a0e366a4db526858fcff0e5584e20be/.github/workflows/node.js.yml#L44